### DISCUSSION TITLE
Can I use owncast as an RMTP relay?

### DISCUSSION THREAD

**Discussion by leetfin** [owncast/owncast#2170] [Q&A] [answered]
Can I set up so I stream to owncast, and then owncast forwards the video to multiple other RMTP locations? Thanks!

**Comment by gabek** [comment, marked as answer]
No, that's a completely different piece of software you're looking for. Take a look at [Prism](https://github.com/muesli/prism).

**Reply by leetfin** [reply]
Thanks!
